### PR TITLE
Add a contact Form

### PR DESCRIPTION
**Pull request by Katended**
The following have been added:

- Input validation for email, name and message
- Formpree action url

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -213,6 +213,16 @@ <h2>About Myself</h2>
...
+<section class="contact">   
+  <h1 >Contact me</h1>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+  <form action="https://formspree.io/f/mqkoywbg" method="POST" class="pure-form" name="">            
+      <input type="text" class="input-rounded" placeholder="Lorem Ipsum" maxlength="30" name="name" required>                          
+      <input  type="email" class="input-rounded" placeholder="loremipsum@gmail.com" required>             
+      <textarea name="message" maxlength="500" placeholder="Write your message here"  required></textarea>
+      <button type="submit" class="call-to-action">Get in touch</button>
+  </form>
+</section>
```
Comment (leonardodiasb) [inline Index.html:226]: - [ ] Your form is working fine and has a similar UI from the Figma template. But at the bottom, the form section seems to end after the `textarea` and the button seems to stay out of it, in an area with a grey background color. Kindly, make the contact section end at the bottom of the screen, and make sure that the buttom seems to stay inside the contact section which has a purple background color.
 - Your design:
 - 
image

- Figma template:
- 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Katended, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Form working fine. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — APPROVED** [review]
Hi @Katended 👋👋, 

INVALID


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have contributions from your learning partner🥲 don't forget, this is a pair programming activity and the contribution of your partner is also required😉 this was omitted by the first reviewer but it is a requirement😉 Kindly reach out to your learning partner and work with him👍

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Ibizugbe** [comment]
Hello @Katended   I think I stated the issue in the comment let me quote it here again ☺️

`You have submitted a project, that doesn't have contributions from your learning partner🥲 don't forget, this is a pair programming activity and the contribution of your partner is also required😉 this was omitted by the first reviewer but it is a requirement😉 Kindly reach out to your learning partner and work with him👍`

We require you to work with your pair programming partner for this milestone. From your commit message, it's seems you haven't done that. The previous reviewer should have pointed that but probably didn't take note of it.. but this is a compulsory requirement.👍

**Comment by Katended** [comment]
> Hello @Katended I think I stated the issue in the comment let me quote it here again ☺️
> 
> `You have submitted a project, that doesn't have contributions from your learning partner🥲 don't forget, this is a pair programming activity and the contribution of your partner is also required😉 this was omitted by the first reviewer but it is a requirement😉 Kindly reach out to your learning partner and work with him👍`
> 
> We require you to work with your pair programming partner for this milestone. From your commit message, it's seems you haven't done that. The previous reviewer should have pointed that but probably didn't take note of it.. but this is a compulsory requirement.👍
> 
> Hope you're clear now?🤔

@Ibizugbe Ok sure thanks.

**Comment by Ibizugbe** [comment]
Also @Katended , I noticed I made use of the `approve` feature to drop this comment, probably this may have caused a bit of confusion. but I sincerely apologise as that was a mistake on my part🥲 I should have dropped a normal comment or use the `comment` feature. But I am you understand now👍

Thanks for understanding and Happy coding👍

**Review by NahnahAJ — COMMENTED** [review]
## Project Status: INVALIDATED :red_circle: 
Hi @Katended , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode) and as a result, needs to have commits from both you and your partner or partners just as the previous reviewer mentioned. :wink: 
- However, I noticed that you submitted the project again with only commits from you. This is the first collaboration milestone for you so please reach out to your partner and ensure that he or she adds commits to the project thereby collaborating in the project :100: 
- If for some reason you had to work on the project alone, then please add a screenshot of the email you received from student success stating that you can work alone so the next reviewer can proceed to review your project :wink: 
- Your commit history :thinking: 
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by jerryowusu — APPROVED** [review]
Hi @Team 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Form working fine :heavy_check_mark: 
Used Github flow :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**